### PR TITLE
[FIX] avoid in-place replacements

### PR DESCRIPTION
**Pull request by skoudoro**
This is a follow of @MarcCote comments as you can see [here](https://github.com/fury-gl/fury/pull/190#discussion_r393198045).

Thank you for the feedback! Can you merge this PR when CI's are done and if you do not see anything else.

Thank you

### CODE REVIEW

**Comment by MarcCote** [comment]
LGTM.

File: fury/ui.py [hunk 1]
```diff
@@ -3349,26 +3350,26 @@ def __init__(self, labels, checked_labels=None, padding=1, font_size=18,
...
-        checked_labels: list(string), optional
-            List of labels that are checked on setting up.
         """
 
         self.labels = list(reversed(labels))
         self._padding = padding
         self._font_size = font_size
         self.font_family = font_family
-        self.checked_labels = checked_labels or []
+        self.checked_labels = list(checked_labels) or []
```
Comment (MarcCote) [inline fury/ui.py]: Oops, I spoke too fast. Just realized this `list(checked_labels)` will fail is it is `None` (i.e.  the default).

Comment (MarcCote) [reply fury/ui.py]: Better to just do `self.checked_labels = list(checked_labels)` and change the default to `[]` as I suggested. Calling `list` should do a soft copy of the list.

Comment (skoudoro) [reply fury/ui.py]: right! Why I did not see it too....  So maybe I should just replace `None` by `()`. What do you think?

Comment (MarcCote) [reply fury/ui.py]: That's even better ^.

Comment (MarcCote) [reply fury/ui.py]: and `self.checked_labels = list(checked_labels)` right?

Comment (MarcCote) [reply fury/ui.py]: No need for the `... or ...`

Comment (skoudoro) [reply fury/ui.py]: yes, I will update that now

**Pull request merged**